src/parent/services/LayoutsService.js

```diff
@@ -0,0 +1,70 @@
+import * as Layouts from 'openfin-layouts';
+
+const uuid = fin.desktop.Application.getCurrent().uuid;
+
+export function start(store) {
+    const layoutsStore = LayoutsStore();
+
+    fin.desktop.InterApplicationBus.subscribe(uuid, 'join-snap-group', updateGroupsAndNotify);
+    fin.desktop.InterApplicationBus.subscribe(uuid, 'leave-snap-group', updateGroupsAndNotify);
+
+    store.subscribe(updateGroupsAndNotify);
```

Comment:
Took me a while to find this function -- would've been easier to locate if this were below the function declaration